Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM alpine:3.8
+
+ENV LOCUST_VERSION=0.9.0 \
+    LOCUST_USER=locust \
+    LOCUST_UID=10000 \
+    LOCUST_GROUP=locust \
+    LOCUST_GID=10000 \
+    LOCUST_HOME=/opt/locust \
+    LANG=C.UTF-8 \
+    LE_STAGING_URL=https://letsencrypt.org/certs/fakelerootx1.pem \
+    LE_STAGING_SHA=12ee1647b9e344a110771b98ebce1f245dda0cd1 \
+    LE_STAGING_FILE=/usr/local/share/ca-certificates/fakelerootx1.pem
+
+RUN apk add --update --no-cache \
+      ca-certificates \
+      openssl \
+      py3-zmq \
+      python3 \
+    && apk add --no-cache --update --virtual build-dependencies \
+      gcc \
+      musl-dev \
+      python3-dev \
+    && pip3 install locustio=="${LOCUST_VERSION}" \
+    && mkdir -p "${LOCUST_HOME}" \
+    && addgroup -g "${LOCUST_GID}" "${LOCUST_GROUP}" \
+    && adduser -g "Kafka user" -D -h "${LOCUST_HOME}" -G "${LOCUST_GROUP}" -s /sbin/nologin -u "${LOCUST_UID}" "${LOCUST_USER}" \
```

Comment:
Kafka user? :laughing:

Reply:
Also Alpine comes with `nobody` and `guest` user, perhaps you can use one of them to save some lines of code. https://git.alpinelinux.org/aports/tree/main/alpine-baselayout/passwd#n27

Reply:
I agree with a dedicated user vs overloading generic users.